src/field.rs

```diff
@@ -50,7 +50,7 @@ pub struct Element<F: Field> {
     /// F needs to be present in a struct field, otherwise the compiler will complain that it is
     /// unused. In reality it is used, but only at compile time. For example, some functions take an
     /// `Element<F>` and call `F::order()`.
-    phantom: PhantomData<F>,
+    phantom: PhantomData<*const F>,
```

Comment:
`PhantomData<*const F>` is the preferred declaration according to the documentation.